### ISSUE TITLE
Improve accessibility of images managed in VA.gov CMS (alt text and other measures)

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#6862]
### User Story or Problem Statement

As a screen reader user, I want to hear the text equivalent for each image conveying information so that I don’t miss any information on the page.

### Affected users and stakeholders

* Veterans and other users of VA.gov
* Editors of all products with images.

### Hypothesis

We believe that we can greatly improve the accessibility of existing and future images managed in the CMS

We'll know this to be true once an accessiblity expert has reviewed the results of an image alt text audit, and that we have moved the needle on % of images with useful alt text.  

We believe that improved help text, training, and possibly some automated tools will help move that needle, but we need additional research to determine what will be most affective given existing resources

## Acceptance Criteria
- [ ] TBD

## Design principles

Veteran-centered
- [ ] `Single source of truth`: Increase reliability and consistency of content on VA.gov by providing a single source of truth.
- [x] `Accessible, plain language`: Provide guardrails and guidelines to ensure content quality.
- [ ] `Purposely structured content`: Ensure Content API can deliver content whose meaning matches its structure.
- [ ] `Content lifecycle governance`: Produce tools, processes and policies to maintain content quality throughout its lifecycle.

Editor-centered
- [ ] `Purpose-driven`: Create an opportunity to involve the editor community in VA’s mission and content strategy goals.
- [ ] `Efficient`: Remove distractions and create clear, straightforward paths to get the job done.
- [ ] `Approachable`: Offer friendly guidance over authoritative instruction.
- [ ] `Consistent`: Reduce user’s mental load by allowing them to fall back on pattern recognition to complete tasks.
- [ ] `Empowering`: Provide clear information to help editors make decisions about their work.

### CMS Team

Please leave only the team that will do this work selected. If you're not sure, it's fine to leave both selected.

- [ ] `Platform CMS Team`
- [x] `Sitewide CMS Team`
  - [x] `⭐️ Content ops`
  - [ ] `⭐️ CMS experience`
  - [ ] `⭐️ Offices`
  - [ ] `⭐️ Product support`
  - [ ] `⭐️ User support`

**Comment by EWashb** [comment]
@laflannery how do you feel about this epic and the associated task? I feel like we have been making strides outside of this original epic to improve image accessibility, and therefore, this is outdated/should be closed.

**Comment by laflannery** [comment]
I agree, we have a new/different direction we are pursuing
